migrator/cf.py

```diff
@@ -54,9 +58,19 @@ def create_bare_migrator_service_instance_in_space(
     return {
         "guid": response["metadata"]["guid"],
         "state": response["entity"]["last_operation"]["state"],
+        "type": response["entity"]["last_operation"]["type"],
     }
 
 
 def get_migrator_service_instance_status(instance_id, client):
     response = client.v2.service_instances.get(instance_id)
     return response["entity"]["last_operation"]["state"]
+
+
+def update_existing_cdn_domain_service_instance(instance_id, params, client):
+    response = client.v2.service_instances.update(instance_id, parameters=params)
+    return {
```

Comment:
I think we should probably just return the raw response here

Reply:
Fair enough!  Should we do the same with the `create` method, or just for this `update` method?

Reply:
Raw response now returned!